### PR TITLE
feat: Support HTTP handler

### PR DESCRIPTION
**Pull request by Davidmattei**
|Q              |A  |
|---------------|---|
|Bug fix?       |n|
|New feature?   |y|	
|BC breaks?     |n|	
|Deprecations?  |n|	
|Fixed tickets  |n|

### CODE REVIEW

File: src/Config/Config.php [hunk 1]
```diff
@@ -22,8 +22,18 @@ public function getEndpoint(): string
         return $this->endpoint;
     }
 
+    public function getEndpointJson(): array
```
Comment (dameert) [inline src/Config/Config.php]: strange naming convention "json" to return an array ?

Comment (dameert) [reply src/Config/Config.php]: maybe this should be getEndpoint and the other funcion getEndpointRaw ?
Or call this function getEndpointDecoded

File: src/Handler/HttpHandler.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    public function handle(HandleRequestInterface $handleRequest): HandleResponseInterface
+    {
+        try {
+            $config = $this->configFactory->create($handleRequest);
+            $httpRequest = new HttpRequest($config);
+
+
+            return new HttpHandleResponse();
```
Comment (dameert) [inline src/Handler/HttpHandler.php]: Isn't there something missing here?, Oh I see the WIP status 👍

**Review by dameert — CHANGES_REQUESTED** [review]

File: src/Config/Config.php (hunk 1 above)
Comment (Davidmattei) [reply src/Config/Config.php]: Renamed it too getEndpointFromJson maybe in the feature we create endpoint from yaml, xml, php :)

File: src/Config/Config.php [hunk 3]
```diff
@@ -22,8 +22,24 @@ public function getEndpoint(): string
...
     {
         return $this->message;
     }
+
+    /**
+     * @return array<mixed>
+     */
+    public function getMessageFromJson(): array
+    {
+        return json_decode($this->message, true) ?? [];
```
Comment (theus77) [inline src/Config/Config.php]: \json_decode

File: src/Config/Config.php [hunk 4]
```diff
@@ -22,8 +22,24 @@ public function getEndpoint(): string
         return $this->endpoint;
     }
 
+    /**
+     * @return array<mixed>
+     */
+    public function getEndpointFromJson(): array
+    {
+        return json_decode($this->endpoint, true) ?? [];
```
Comment (theus77) [inline src/Config/Config.php]: \json_decode

Comment (Davidmattei) [reply src/Config/Config.php]: Thx good catch https://github.com/ems-project/EMSSubmissionBundle/pull/42/commits/4f191e8f7f6132f6d655b40a8e6e36d566e36a5c

File: src/Config/Config.php (hunk 3 above)
Comment (Davidmattei) [reply src/Config/Config.php]: https://github.com/ems-project/EMSSubmissionBundle/pull/42/commits/4f191e8f7f6132f6d655b40a8e6e36d566e36a5c

**Review by dameert — APPROVED** [review]

**Pull request merged**